src/js/popup/popup.js

```diff
@@ -464,10 +464,17 @@ async function showRelayPanel(tipPanelToShow) {
   const { relayAddresses, maxNumAliases } = await getRemainingAliases();
   const numRemaining = maxNumAliases - relayAddresses.length;
   const remainingAliasMessage = document.querySelector(".aliases-remaining");
-  remainingAliasMessage.textContent = browser.i18n.getMessage("popupRemainingAliases_2_mask", [numRemaining, maxNumAliases]);
   const getUnlimitedAliases = document.querySelector(".premium-cta");
   getUnlimitedAliases.textContent = browser.i18n.getMessage("popupGetUnlimitedAliases_mask");
   document.body.classList.add("relay-panel");
+  
+  // Prevent negative masks from showing, default to 0 if all free masks have been used up
+  let positiveNumRemaining = numRemaining;
+  if (numRemaining <= 0) {
+    positiveNumRemaining = 0;
```

Comment:
Alternatively, console log the `positiveNumRemaining` variable for a premium account with over 5 aliases.